src/clj/rems/email/template.clj

```diff
@@ -193,3 +193,20 @@
                (text :t.email/footer))}])))
 
 ;; TODO member-joined?
+
+(defn handler-reminder-email [lang handler applications]
+  (with-language lang
```

Comment:
The function emails-to-recipients above does the language selection like this:

`(with-language (:language (user-settings/get-user-settings (:userid recipient)))`

Maybe this function could follow a similar template.